tika/server.go

```diff
@@ -151,9 +132,9 @@ func (s *Server) Start(ctx context.Context) error {
 	//Check to see if there are any tuples in the servers java props, if so
 	//format and add to the props variable
 
-	if len(s.javaprops) > 0 {
-		for i := range s.javaprops {
-			props = fmt.Sprintf("-D%s=%s ", i, s.javaprops[i])
+	if len(s.Javaprops) > 0 {
+		for i := range s.Javaprops {
+			props = fmt.Sprintf("-D%s=%s ", i, s.Javaprops[i])
```

Comment:
This overwrites props on every iteration. Perhaps something like this?

```go
props := []string{}
for k, v := range s.JavaProps {
	props = append(props, fmt.Sprintf("-D%s=%s", k, v))
}
propsArgs := strings.Join(props, " ")
```

(Also gets the key and value from the `range`, rather than just the key.)